File: app/src/test/java/com/example/moviemvp/movielist/MovieListModelTest.java
package com.example.moviemvp.movielist;

import org.junit.Before;
import org.junit.Test;
import org.mockito.Mockito;
import org.mockito.MockitoAnnotations;

public class MovieListModelTest {
    private MovieListModel model;
    private MovieListContract.Model.OnFinishedListener a;

    @Before
    public void setUp() {
        MockitoAnnotations.initMocks(this);
        model = new MovieListModel();
    }

    @Test
    public void getMovieList() {
        model.getMovieList(a, Mockito.eq(1));

    }
}

File: app/src/test/java/com/example/moviemvp/movie_list/MovieListModelTest.kt
package com.example.moviemvp.movie_list

import com.example.moviemvp.movie_list.MovieListContract.Model.OnFinishedListener
import org.junit.Before
import org.junit.Test

import org.junit.Assert.*
import org.mockito.Mock
import org.mockito.Mockito
import org.mockito.MockitoAnnotations

 class MovieListModelTest {

    private lateinit var model: MovieListModel
    private lateinit var listener: OnFinishedListener

    @Before
    fun setUp() {

        MockitoAnnotations.initMocks(this)
        model= MovieListModel()
        listener=Mockito.mock(OnFinishedListener::class.java)
    }

    @Test
    fun getMovieList() {

        model.getMovieList(listener, Mockito.eq(1))
    }
}